src/LCT.PackageIdentifier/BomCreator.cs

```diff
@@ -165,5 +166,27 @@ private async Task<Bom> ComponentIdentification(CommonAppSettings appSettings, I
             }
             return bom;
         }
+
+        public async Task<bool> CheckJFrogConnection()
+        {
+            var response = await JFrogService.CheckJFrogConnectivity();
+            if (response != null)
+            {
+                if (response.IsSuccessStatusCode)
+                {
+                    Logger.Logger.Log(null, Level.Warn, $"JFrog Connection was successfull!!", null);
```

Comment:
for success shouldn't it be displayed as green color, and with Level.Notice ?

Reply:
Changed